commands-yml/commands/session/get.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: Get Session Capabilities
+short_description: Retrieve the capabilities of the specified session
+   
+example_usage:
+  java:
+    |
+        DriverSessions driverSessions = new DriverSessions();
+        Session session = driverSessions.get("c8db88a0-47a6-47a1-802d-164d746c06aa");
```

Comment:
AFAIK, to get capabilities of a session it should be `driver.getCapabilities()` or from `driver.getSessionDetails()`. `DriverSessions` may not work in any of mobile specific drivers or i don't recommend this way to users.